docs/source/contributor-guide/index.md

```diff
@@ -134,6 +134,10 @@ cargo test -p datafusion --tests sql_integration
 
 One very important test is the [sql_integration](../../../datafusion/core/tests/sql_integration.rs) test which validates DataFusion's ability to run a large assortment of SQL queries against an assortment of data setups.
 
+### sqllogictests Tests
```

Comment:
How about adding some comparison with `Rust Integration Tests` and encouraging adding logictestes as far as possible?